CHANGELOG.md

```diff
@@ -39,6 +40,189 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+# 2.0.0-rc1
+
+apisix-ingress-controller 2.0.0-rc1
+
+Welcome to the 2.0.0-rc1 release of apisix-ingress-controller!
+
+This is a feature release.
+
+## Highlights
+
+This release contains following new features:
+- feat: add skip_mtls_uri_regex support for ApisixTls 
```

Comment:
Any changes about the new implementations of the Ingress Controller and CRD?

Reply:
I manually added more instructions, please review again.